src/react-styleguide/src/components/Buttons/Button.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+
```

Comment:
Could declare all prop values:

```
const sizes = ['large', 'small', cta']
const types = ['success', 'warning', 'danger', 'primary', default']
const options = ['outline']
```

This would let you dynamically use proptypes and examples, so your proptypes and examples are always kept up to date with the source code automatically. See inline examples below.